2024-12-15 21:51:02,911:DEBUG:certbot._internal.main:certbot version: 2.9.0
2024-12-15 21:51:02,912:DEBUG:certbot._internal.main:Location of certbot entry point: /home/runner/JobSeekrBot/.pythonlibs/bin/certbot
2024-12-15 21:51:02,912:DEBUG:certbot._internal.main:Arguments: ['--webroot', '--webroot-path', '/home/runner/JobSeekrBot/static/.well-known', '--email', 'admin@hostme.co.il', '--agree-tos', '--no-eff-email', '-d', 'aim.jobai.tech', '--config-dir', '/home/runner/JobSeekrBot/ssl/letsencrypt', '--work-dir', '/home/runner/JobSeekrBot/ssl/letsencrypt/work', '--logs-dir', '/home/runner/JobSeekrBot/ssl/letsencrypt/logs', '--non-interactive']
2024-12-15 21:51:02,912:DEBUG:certbot._internal.main:Discovered plugins: PluginsRegistry(PluginEntryPoint#manual,PluginEntryPoint#null,PluginEntryPoint#standalone,PluginEntryPoint#webroot)
2024-12-15 21:51:02,927:DEBUG:certbot._internal.log:Root logging level set at 30
2024-12-15 21:51:02,928:DEBUG:certbot._internal.plugins.selection:Requested authenticator webroot and installer None
2024-12-15 21:51:02,928:DEBUG:certbot._internal.plugins.selection:Single candidate plugin: * webroot
Description: Saves the necessary validation files to a .well-known/acme-challenge/ directory within the nominated webroot path. A seperate HTTP server must be running and serving files from the webroot path. HTTP challenge only (wildcards not supported).
Interfaces: Authenticator, Plugin
Entry point: EntryPoint(name='webroot', value='certbot._internal.plugins.webroot:Authenticator', group='certbot.plugins')
Initialized: <certbot._internal.plugins.webroot.Authenticator object at 0x7f8714fa4890>
Prep: True
2024-12-15 21:51:02,928:DEBUG:certbot._internal.plugins.selection:Selected authenticator <certbot._internal.plugins.webroot.Authenticator object at 0x7f8714fa4890> and installer None
2024-12-15 21:51:02,928:INFO:certbot._internal.plugins.selection:Plugins selected: Authenticator webroot, Installer None
2024-12-15 21:51:02,988:DEBUG:certbot._internal.main:Picked account: <Account(RegistrationResource(body=Registration(key=None, contact=(), agreement=None, status=None, terms_of_service_agreed=None, only_return_existing=None, external_account_binding=None), uri='https://acme-v02.api.letsencrypt.org/acme/acct/2115047605', new_authzr_uri=None, terms_of_service=None), de8f11d49b456ead263fc25afeacddd7, Meta(creation_dt=datetime.datetime(2024, 12, 15, 19, 53, 51, tzinfo=<UTC>), creation_host='d706af8e75de', register_to_eff=None))>
2024-12-15 21:51:02,990:DEBUG:acme.client:Sending GET request to https://acme-v02.api.letsencrypt.org/directory.
2024-12-15 21:51:02,994:DEBUG:urllib3.connectionpool:Starting new HTTPS connection (1): acme-v02.api.letsencrypt.org:443
2024-12-15 21:51:03,245:DEBUG:urllib3.connectionpool:https://acme-v02.api.letsencrypt.org:443 "GET /directory HTTP/1.1" 200 746
2024-12-15 21:51:03,247:DEBUG:acme.client:Received response:
HTTP 200
Server: nginx
Date: Sun, 15 Dec 2024 21:51:03 GMT
Content-Type: application/json
Content-Length: 746
Connection: keep-alive
Cache-Control: public, max-age=0, no-cache
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800

{
  "7TZZ0MnsAb0": "https://community.letsencrypt.org/t/adding-random-entries-to-the-directory/33417",
  "keyChange": "https://acme-v02.api.letsencrypt.org/acme/key-change",
  "meta": {
    "caaIdentities": [
      "letsencrypt.org"
    ],
    "termsOfService": "https://letsencrypt.org/documents/LE-SA-v1.4-April-3-2024.pdf",
    "website": "https://letsencrypt.org"
  },
  "newAccount": "https://acme-v02.api.letsencrypt.org/acme/new-acct",
  "newNonce": "https://acme-v02.api.letsencrypt.org/acme/new-nonce",
  "newOrder": "https://acme-v02.api.letsencrypt.org/acme/new-order",
  "renewalInfo": "https://acme-v02.api.letsencrypt.org/draft-ietf-acme-ari-03/renewalInfo",
  "revokeCert": "https://acme-v02.api.letsencrypt.org/acme/revoke-cert"
}
2024-12-15 21:51:03,249:DEBUG:certbot._internal.display.obj:Notifying user: Requesting a certificate for aim.jobai.tech
2024-12-15 21:51:03,257:DEBUG:acme.client:Requesting fresh nonce
2024-12-15 21:51:03,258:DEBUG:acme.client:Sending HEAD request to https://acme-v02.api.letsencrypt.org/acme/new-nonce.
2024-12-15 21:51:03,316:DEBUG:urllib3.connectionpool:https://acme-v02.api.letsencrypt.org:443 "HEAD /acme/new-nonce HTTP/1.1" 200 0
2024-12-15 21:51:03,318:DEBUG:acme.client:Received response:
HTTP 200
Server: nginx
Date: Sun, 15 Dec 2024 21:51:03 GMT
Connection: keep-alive
Cache-Control: public, max-age=0, no-cache
Link: <https://acme-v02.api.letsencrypt.org/directory>;rel="index"
Replay-Nonce: WVWTDxmJz5nZjf0Ffc2gByV40FMtZWcd4D8k9ZpRYdC6H5ls9AU
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800


2024-12-15 21:51:03,318:DEBUG:acme.client:Storing nonce: WVWTDxmJz5nZjf0Ffc2gByV40FMtZWcd4D8k9ZpRYdC6H5ls9AU
2024-12-15 21:51:03,319:DEBUG:acme.client:JWS payload:
b'{\n  "identifiers": [\n    {\n      "type": "dns",\n      "value": "aim.jobai.tech"\n    }\n  ]\n}'
2024-12-15 21:51:03,322:DEBUG:acme.client:Sending POST request to https://acme-v02.api.letsencrypt.org/acme/new-order:
{
  "protected": "eyJhbGciOiAiUlMyNTYiLCAia2lkIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2FjY3QvMjExNTA0NzYwNSIsICJub25jZSI6ICJXVldURHhtSno1blpqZjBGZmMyZ0J5VjQwRk10WldjZDREOGs5WnBSWWRDNkg1bHM5QVUiLCAidXJsIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL25ldy1vcmRlciJ9",
  "signature": "MP46ortNa0F1kSmXLs0DV7fW1TIgXqbsDILtarcURFwbZUu-XpCXeBT6giTSWTD_BBY6MAfqNfQX3Yrb50LfzI2jSKcXf9RPYRUIFirEfw380Fa_UcUACFpQmMrWZMgqRvT8-EHtIz09E8zNlQIEOYz-D_Z9rQOtEfinruIwkJ06YhIj9ruaMi9zNyktLrgpBRf46zs4lIq8CQr_YLFCaSG07JqLxcQBh54YTO0hZWqK3gb654Lx_4Wzr7c6J5SvynqkWBudTrZpXZ59FTLuAh3SRTn5_2T_MiAzErf5dDMcQKNVsrOdgykkV4NMPNxp-WeFp8nwAQ7GWDbx1ypNfw",
  "payload": "ewogICJpZGVudGlmaWVycyI6IFsKICAgIHsKICAgICAgInR5cGUiOiAiZG5zIiwKICAgICAgInZhbHVlIjogImFpbS5qb2JhaS50ZWNoIgogICAgfQogIF0KfQ"
}
2024-12-15 21:51:03,434:DEBUG:urllib3.connectionpool:https://acme-v02.api.letsencrypt.org:443 "POST /acme/new-order HTTP/1.1" 201 348
2024-12-15 21:51:03,434:DEBUG:acme.client:Received response:
HTTP 201
Server: nginx
Date: Sun, 15 Dec 2024 21:51:03 GMT
Content-Type: application/json
Content-Length: 348
Connection: keep-alive
Boulder-Requester: 2115047605
Cache-Control: public, max-age=0, no-cache
Link: <https://acme-v02.api.letsencrypt.org/directory>;rel="index"
Location: https://acme-v02.api.letsencrypt.org/acme/order/2115047605/333690006015
Replay-Nonce: WVWTDxmJ1IMXEXfqKvdhqqjTqKJclNpv5vvliNDXI1kvglPlrpU
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800

{
  "status": "pending",
  "expires": "2024-12-22T21:49:38Z",
  "identifiers": [
    {
      "type": "dns",
      "value": "aim.jobai.tech"
    }
  ],
  "authorizations": [
    "https://acme-v02.api.letsencrypt.org/acme/authz/2115047605/445430450455"
  ],
  "finalize": "https://acme-v02.api.letsencrypt.org/acme/finalize/2115047605/333690006015"
}
2024-12-15 21:51:03,435:DEBUG:acme.client:Storing nonce: WVWTDxmJ1IMXEXfqKvdhqqjTqKJclNpv5vvliNDXI1kvglPlrpU
2024-12-15 21:51:03,435:DEBUG:acme.client:JWS payload:
b''
2024-12-15 21:51:03,437:DEBUG:acme.client:Sending POST request to https://acme-v02.api.letsencrypt.org/acme/authz/2115047605/445430450455:
{
  "protected": "eyJhbGciOiAiUlMyNTYiLCAia2lkIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2FjY3QvMjExNTA0NzYwNSIsICJub25jZSI6ICJXVldURHhtSjFJTVhFWGZxS3ZkaHFxalRxS0pjbE5wdjV2dmxpTkRYSTFrdmdsUGxycFUiLCAidXJsIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2F1dGh6LzIxMTUwNDc2MDUvNDQ1NDMwNDUwNDU1In0",
  "signature": "bVSkfg7ktLXpBVfo9_UPPLWB9j11L528_KHu5JrTohjtamfi7HUPi9QQpwuyGkboGh1Gd-CHAnn8MprdTIKpJdY8_yYiBR24qveyPnU0IPLrgvdY_PlooHHS_wRY79on05M4OP8JgCXFtqbANu1GZYYlRVy3IJPlSK7Q6h_czKHeIrF1AluPfksxl3ev7RoKKwwiG1mMAzFKgkt8XCYtq2lfX2AOTim2YlpeGUKWplPdYakt3YbgbbCY1F5hynNCYKMS25fjKjVeIK5VLE0SnoYB0PGvgoQmacTUUrezan8gawiD0YIHqFRJZ5JpiAJMQC5YSojzBqQ-Ni9X04jQNw",
  "payload": ""
}
2024-12-15 21:51:03,495:DEBUG:urllib3.connectionpool:https://acme-v02.api.letsencrypt.org:443 "POST /acme/authz/2115047605/445430450455 HTTP/1.1" 200 822
2024-12-15 21:51:03,496:DEBUG:acme.client:Received response:
HTTP 200
Server: nginx
Date: Sun, 15 Dec 2024 21:51:03 GMT
Content-Type: application/json
Content-Length: 822
Connection: keep-alive
Boulder-Requester: 2115047605
Cache-Control: public, max-age=0, no-cache
Link: <https://acme-v02.api.letsencrypt.org/directory>;rel="index"
Replay-Nonce: WVWTDxmJdjvfu1f5OPEYY7oycm7r9LYgPEcK4G8L-1jgrmhAhU4
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800

{
  "identifier": {
    "type": "dns",
    "value": "aim.jobai.tech"
  },
  "status": "pending",
  "expires": "2024-12-22T21:49:38Z",
  "challenges": [
    {
      "type": "dns-01",
      "url": "https://acme-v02.api.letsencrypt.org/acme/chall/2115047605/445430450455/LMhrOQ",
      "status": "pending",
      "token": "hHNxPQXpIJmerFhRluR-2fd3lnSNPFitGHLtgRKjYiw"
    },
    {
      "type": "http-01",
      "url": "https://acme-v02.api.letsencrypt.org/acme/chall/2115047605/445430450455/alGYLA",
      "status": "pending",
      "token": "hHNxPQXpIJmerFhRluR-2fd3lnSNPFitGHLtgRKjYiw"
    },
    {
      "type": "tls-alpn-01",
      "url": "https://acme-v02.api.letsencrypt.org/acme/chall/2115047605/445430450455/ecakqg",
      "status": "pending",
      "token": "hHNxPQXpIJmerFhRluR-2fd3lnSNPFitGHLtgRKjYiw"
    }
  ]
}
2024-12-15 21:51:03,497:DEBUG:acme.client:Storing nonce: WVWTDxmJdjvfu1f5OPEYY7oycm7r9LYgPEcK4G8L-1jgrmhAhU4
2024-12-15 21:51:03,497:INFO:certbot._internal.auth_handler:Performing the following challenges:
2024-12-15 21:51:03,498:INFO:certbot._internal.auth_handler:http-01 challenge for aim.jobai.tech
2024-12-15 21:51:03,498:INFO:certbot._internal.plugins.webroot:Using the webroot path /home/runner/JobSeekrBot/static/.well-known for all unmatched domains.
2024-12-15 21:51:03,498:DEBUG:certbot._internal.plugins.webroot:Creating root challenges validation dir at /home/runner/JobSeekrBot/static/.well-known/.well-known/acme-challenge
2024-12-15 21:51:03,504:DEBUG:certbot._internal.plugins.webroot:Attempting to save validation to /home/runner/JobSeekrBot/static/.well-known/.well-known/acme-challenge/hHNxPQXpIJmerFhRluR-2fd3lnSNPFitGHLtgRKjYiw
2024-12-15 21:51:03,505:DEBUG:acme.client:JWS payload:
b'{}'
2024-12-15 21:51:03,507:DEBUG:acme.client:Sending POST request to https://acme-v02.api.letsencrypt.org/acme/chall/2115047605/445430450455/alGYLA:
{
  "protected": "eyJhbGciOiAiUlMyNTYiLCAia2lkIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2FjY3QvMjExNTA0NzYwNSIsICJub25jZSI6ICJXVldURHhtSmRqdmZ1MWY1T1BFWVk3b3ljbTdyOUxZZ1BFY0s0RzhMLTFqZ3JtaEFoVTQiLCAidXJsIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2NoYWxsLzIxMTUwNDc2MDUvNDQ1NDMwNDUwNDU1L2FsR1lMQSJ9",
  "signature": "BgS2vS66sPVlOh7F72LXrjirXrXJACmHwHq0OmuPXxIfJa38KUYuHGKpfLAHZY-QcEadCk0uGnQZZq8GdMv717qCFw1hQTuT8z4HzNXAUjsB-FOqDI0tEHFpghsZWx3xyjzu-RFR_Q8cUdr5mfhsegXozA1MF9LN8KBLOoMkqn1EUWazjeZUi1AskVkXQgJhsIL7aQ_cBOyNatOjh-zUqS3f1vaGx2uT2vXSzsUkGs2xN-QNdf11cXV2gieNAUCt779qFWgxS4TrN1Jx5lRfWAgrK6OeKWHXFHlC2d_ewW3PFY1E-20kAiHdLzuIavVBkoYpVCLhdQVLA6BW_0A6bg",
  "payload": "e30"
}
2024-12-15 21:51:03,570:DEBUG:urllib3.connectionpool:https://acme-v02.api.letsencrypt.org:443 "POST /acme/chall/2115047605/445430450455/alGYLA HTTP/1.1" 200 195
2024-12-15 21:51:03,570:DEBUG:acme.client:Received response:
HTTP 200
Server: nginx
Date: Sun, 15 Dec 2024 21:51:03 GMT
Content-Type: application/json
Content-Length: 195
Connection: keep-alive
Boulder-Requester: 2115047605
Cache-Control: public, max-age=0, no-cache
Link: <https://acme-v02.api.letsencrypt.org/directory>;rel="index", <https://acme-v02.api.letsencrypt.org/acme/authz/2115047605/445430450455>;rel="up"
Location: https://acme-v02.api.letsencrypt.org/acme/chall/2115047605/445430450455/alGYLA
Replay-Nonce: GJdccAF65CVSP9JD4afnle6Xrm2Gt1UVcO_tVg9LTeJP1eMZrCw
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800

{
  "type": "http-01",
  "url": "https://acme-v02.api.letsencrypt.org/acme/chall/2115047605/445430450455/alGYLA",
  "status": "pending",
  "token": "hHNxPQXpIJmerFhRluR-2fd3lnSNPFitGHLtgRKjYiw"
}
2024-12-15 21:51:03,571:DEBUG:acme.client:Storing nonce: GJdccAF65CVSP9JD4afnle6Xrm2Gt1UVcO_tVg9LTeJP1eMZrCw
2024-12-15 21:51:03,571:INFO:certbot._internal.auth_handler:Waiting for verification...
2024-12-15 21:51:04,572:DEBUG:acme.client:JWS payload:
b''
2024-12-15 21:51:04,573:DEBUG:acme.client:Sending POST request to https://acme-v02.api.letsencrypt.org/acme/authz/2115047605/445430450455:
{
  "protected": "eyJhbGciOiAiUlMyNTYiLCAia2lkIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2FjY3QvMjExNTA0NzYwNSIsICJub25jZSI6ICJHSmRjY0FGNjVDVlNQOUpENGFmbmxlNlhybTJHdDFVVmNPX3RWZzlMVGVKUDFlTVpyQ3ciLCAidXJsIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2F1dGh6LzIxMTUwNDc2MDUvNDQ1NDMwNDUwNDU1In0",
  "signature": "Vvv3G9xSr-qur2Dk3sQV7cTrJ3hWktN0QeH5tDLqodXZ2uuP4VKmdUp1CsKAC6Y6mB7YQvBQeGrz_xwH2aBOgZM3W2LWfci2bFRhpLlLzX9WZCs5YV-4dkKOlQ1r3__LonQhn6OKdT9YtBSY6RBgdCOZgOKfdvr-2ucybgKHGY3jGDZpSFL3b6Xh6NMClBcplFlNou_a6GwOdiTEUo5GbjAhRxIWkRJfwIq6lsOdB4xJeey7smFD3mvz0M64vbXUI1ipbbCdtB1arr3Hu4U--n7hj-PfF-TvmtWofABvpi4wKHYA4QxYOqb7nN3AXp5Oqad5po6Asn-6DlZtHYD6Lw",
  "payload": ""
}
2024-12-15 21:51:04,649:DEBUG:urllib3.connectionpool:https://acme-v02.api.letsencrypt.org:443 "POST /acme/authz/2115047605/445430450455 HTTP/1.1" 200 1338
2024-12-15 21:51:04,649:DEBUG:acme.client:Received response:
HTTP 200
Server: nginx
Date: Sun, 15 Dec 2024 21:51:04 GMT
Content-Type: application/json
Content-Length: 1338
Connection: keep-alive
Boulder-Requester: 2115047605
Cache-Control: public, max-age=0, no-cache
Link: <https://acme-v02.api.letsencrypt.org/directory>;rel="index"
Replay-Nonce: GJdccAF6K8-rUZq1-tCj7bqFhZubT0kOhAygfYPg_AOar2-Ezeo
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800

{
  "identifier": {
    "type": "dns",
    "value": "aim.jobai.tech"
  },
  "status": "invalid",
  "expires": "2024-12-22T21:49:38Z",
  "challenges": [
    {
      "type": "http-01",
      "url": "https://acme-v02.api.letsencrypt.org/acme/chall/2115047605/445430450455/alGYLA",
      "status": "invalid",
      "validated": "2024-12-15T21:51:03Z",
      "error": {
        "type": "urn:ietf:params:acme:error:unauthorized",
        "detail": "34.82.58.13: Invalid response from https://aim.jobai.tech/.well-known/acme-challenge/hHNxPQXpIJmerFhRluR-2fd3lnSNPFitGHLtgRKjYiw: 404",
        "status": 403
      },
      "token": "hHNxPQXpIJmerFhRluR-2fd3lnSNPFitGHLtgRKjYiw",
      "validationRecord": [
        {
          "url": "http://aim.jobai.tech/.well-known/acme-challenge/hHNxPQXpIJmerFhRluR-2fd3lnSNPFitGHLtgRKjYiw",
          "hostname": "aim.jobai.tech",
          "port": "80",
          "addressesResolved": [
            "34.82.58.13"
          ],
          "addressUsed": "34.82.58.13"
        },
        {
          "url": "https://aim.jobai.tech/.well-known/acme-challenge/hHNxPQXpIJmerFhRluR-2fd3lnSNPFitGHLtgRKjYiw",
          "hostname": "aim.jobai.tech",
          "port": "443",
          "addressesResolved": [
            "34.82.58.13"
          ],
          "addressUsed": "34.82.58.13"
        }
      ]
    }
  ]
}
2024-12-15 21:51:04,650:DEBUG:acme.client:Storing nonce: GJdccAF6K8-rUZq1-tCj7bqFhZubT0kOhAygfYPg_AOar2-Ezeo
2024-12-15 21:51:04,650:INFO:certbot._internal.auth_handler:Challenge failed for domain aim.jobai.tech
2024-12-15 21:51:04,650:INFO:certbot._internal.auth_handler:http-01 challenge for aim.jobai.tech
2024-12-15 21:51:04,650:DEBUG:certbot._internal.display.obj:Notifying user: 
Certbot failed to authenticate some domains (authenticator: webroot). The Certificate Authority reported these problems:
  Domain: aim.jobai.tech
  Type:   unauthorized
  Detail: 34.82.58.13: Invalid response from https://aim.jobai.tech/.well-known/acme-challenge/hHNxPQXpIJmerFhRluR-2fd3lnSNPFitGHLtgRKjYiw: 404

Hint: The Certificate Authority failed to download the temporary challenge files created by Certbot. Ensure that the listed domains serve their content from the provided --webroot-path/-w and that files created there can be downloaded from the internet.

2024-12-15 21:51:04,651:DEBUG:certbot._internal.error_handler:Encountered exception:
Traceback (most recent call last):
  File "/nix/store/8pa1a0bd8r0yxlb6f5if6lrl031rgbf2-python3.11-certbot-2.9.0/lib/python3.11/site-packages/certbot/_internal/auth_handler.py", line 108, in handle_authorizations
    self._poll_authorizations(authzrs, max_retries, max_time_mins, best_effort)
  File "/nix/store/8pa1a0bd8r0yxlb6f5if6lrl031rgbf2-python3.11-certbot-2.9.0/lib/python3.11/site-packages/certbot/_internal/auth_handler.py", line 212, in _poll_authorizations
    raise errors.AuthorizationError('Some challenges have failed.')
certbot.errors.AuthorizationError: Some challenges have failed.

2024-12-15 21:51:04,651:DEBUG:certbot._internal.error_handler:Calling registered functions
2024-12-15 21:51:04,651:INFO:certbot._internal.auth_handler:Cleaning up challenges
2024-12-15 21:51:04,651:DEBUG:certbot._internal.plugins.webroot:Removing /home/runner/JobSeekrBot/static/.well-known/.well-known/acme-challenge/hHNxPQXpIJmerFhRluR-2fd3lnSNPFitGHLtgRKjYiw
2024-12-15 21:51:04,651:DEBUG:certbot._internal.plugins.webroot:All challenges cleaned up
2024-12-15 21:51:04,652:DEBUG:certbot._internal.log:Exiting abnormally:
Traceback (most recent call last):
  File "/home/runner/JobSeekrBot/.pythonlibs/bin/certbot", line 8, in <module>
    sys.exit(main())
             ^^^^^^
  File "/nix/store/8pa1a0bd8r0yxlb6f5if6lrl031rgbf2-python3.11-certbot-2.9.0/lib/python3.11/site-packages/certbot/main.py", line 19, in main
    return internal_main.main(cli_args)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/8pa1a0bd8r0yxlb6f5if6lrl031rgbf2-python3.11-certbot-2.9.0/lib/python3.11/site-packages/certbot/_internal/main.py", line 1894, in main
    return config.func(config, plugins)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/8pa1a0bd8r0yxlb6f5if6lrl031rgbf2-python3.11-certbot-2.9.0/lib/python3.11/site-packages/certbot/_internal/main.py", line 1600, in certonly
    lineage = _get_and_save_cert(le_client, config, domains, certname, lineage)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/8pa1a0bd8r0yxlb6f5if6lrl031rgbf2-python3.11-certbot-2.9.0/lib/python3.11/site-packages/certbot/_internal/main.py", line 143, in _get_and_save_cert
    lineage = le_client.obtain_and_enroll_certificate(domains, certname)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/8pa1a0bd8r0yxlb6f5if6lrl031rgbf2-python3.11-certbot-2.9.0/lib/python3.11/site-packages/certbot/_internal/client.py", line 517, in obtain_and_enroll_certificate
    cert, chain, key, _ = self.obtain_certificate(domains)
                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/8pa1a0bd8r0yxlb6f5if6lrl031rgbf2-python3.11-certbot-2.9.0/lib/python3.11/site-packages/certbot/_internal/client.py", line 428, in obtain_certificate
    orderr = self._get_order_and_authorizations(csr.data, self.config.allow_subset_of_names)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/8pa1a0bd8r0yxlb6f5if6lrl031rgbf2-python3.11-certbot-2.9.0/lib/python3.11/site-packages/certbot/_internal/client.py", line 496, in _get_order_and_authorizations
    authzr = self.auth_handler.handle_authorizations(orderr, self.config, best_effort)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nix/store/8pa1a0bd8r0yxlb6f5if6lrl031rgbf2-python3.11-certbot-2.9.0/lib/python3.11/site-packages/certbot/_internal/auth_handler.py", line 108, in handle_authorizations
    self._poll_authorizations(authzrs, max_retries, max_time_mins, best_effort)
  File "/nix/store/8pa1a0bd8r0yxlb6f5if6lrl031rgbf2-python3.11-certbot-2.9.0/lib/python3.11/site-packages/certbot/_internal/auth_handler.py", line 212, in _poll_authorizations
    raise errors.AuthorizationError('Some challenges have failed.')
certbot.errors.AuthorizationError: Some challenges have failed.
2024-12-15 21:51:04,657:ERROR:certbot._internal.log:Some challenges have failed.
